x-pack/solutions/observability/plugins/apm/test/scout/ui/fixtures/page_objects/agent_configurations.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { KibanaUrl, ScoutPage } from '@kbn/scout-oblt';
+import { EuiComboBoxWrapper } from '@kbn/scout-oblt';
+
+export class AgentConfigurationsPage {
+  constructor(private readonly page: ScoutPage, private readonly kbnUrl: KibanaUrl) {}
+
+  async goto() {
+    await this.page.goto(`${this.kbnUrl.app('apm')}/settings/agent-configuration`);
+    await this.page.waitForLoadingIndicatorHidden();
+
+    // Wait for the page content to load
+    await this.page
+      .getByRole('heading', { name: 'Settings', level: 1 })
+      .waitFor({ timeout: 10000 });
```

Comment:
question: do you see issues with running it like `.waitFor();` ?

ideally tests should rely on default timeout (currently: `10_000` ms) and if tests timeout too often on the specific env, we will increase it in centrally as well, but then it won't take affect in such cases.

If default timeout is not 10 sec, something is wrong in central playwright confiugration